### PR TITLE
(kubectl events): Add integration tests

### PR DESCRIPTION
**Pull request by ardaguclu**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
This PR adds integration tests for `kubectl alpha events`

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: test/cmd/events.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    ### Create a new namespace
+    # Pre-condition: the test-events namespace does not exist
+    kube::test::get_object_assert 'namespaces' "{{range.items}}{{ if eq ${id_field:?} \\\"test-events\\\" }}found{{end}}{{end}}:" ':'
+    # Command
+    kubectl create namespace test-events
+    # Post-condition: namespace 'test-jobs' is created.
+    kube::test::get_object_assert 'namespaces/test-events' "{{$id_field}}" 'test-events'
+
+    # Pre-condition: event does not exist for Cronjob/pi in test-events namespace
```
Comment (brianpursley) [inline test/cmd/events.sh:37]: technically, this is using the `-A` flag, so it is not just checking in the `test-events` namespace.  Is that OK?  Or does the comment need to be changed?

**Review by brianpursley — CHANGES_REQUESTED** [review]
Looks good.  Left a few minor comments

File: test/cmd/events.sh (hunk 1 above)
Comment (ardaguclu) [reply test/cmd/events.sh]: I changed the comment. We need to ensure that Cronjob/pi should not exist in any namespace.

**Comment by ardaguclu** [comment]
@brianpursley could you PTAL?. Thanks.

**Review by brianpursley — APPROVED** [review]
/lgtm

**Pull request merged**